app/View/Edit.js

```diff
@@ -1,20 +1,101 @@
+import FormFields from '../Component/Form/FormFields';
 
 class EditView extends React.Component {
+    constructor() {
+        super();
+
+        this.state = EditStore.getState();
+        this.shouldComponentUpdate = shouldComponentUpdate.bind(this);
+        this.save = this.save.bind(this);
+    }
+
+    componentWillMount() {
+        EditStore.addChangeListener(this.onChange.bind(this));
+
+        this.refreshData();
+    }
+
+    componentWillUnmount() {
+        EditStore.removeChangeListener(this.onChange.bind(this));
+    }
+
+    getView(entityName) {
+        entityName = entityName || this.context.router.getCurrentParams().entity;
+
+        return this.props.configuration.getEntity(entityName).editionView();
+    }
+
+    onChange() {
+        this.setState(EditStore.getState());
+    }
+
+    refreshData() {
+        let {id} = this.context.router.getCurrentParams();
+        let {sortField, sortDir} = this.context.router.getCurrentQuery() || {};
+
+        EditActions.loadData(this.props.configuration, this.getView(), id, sortField, sortDir);
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.params.entity !== this.props.params.entity
+            || nextProps.query.sortField !== this.props.query.sortField
+            || nextProps.query.sortDir !== this.props.query.sortDir) {
+
+            this.refreshData();
+        }
+    }
+
+    updateField(name, value) {
+        EditActions.updateData(name, value);
+    }
+
+    save(e) {
+        e.preventDefault();
+
+        EditActions.saveData(this.props.configuration, this.getView());
+    }
+
     render() {
         let params = this.context.router.getCurrentParams(),
             entityName = params.entity,
-            view = this.props.configuration.getEntity(entityName).views["EditView"];
+            view = this.getView(entityName),
+            dataStore = this.state.data.get('dataStore'),
+            values = this.state.data.get('values'),
+            entry = dataStore.getFirstEntry(view.getEntity().uniqueId),
+            actions = view.actions() || ['list', 'delete'];
 
         return (
-            <div className="view list-view">
-                <ViewActions entityName={view.entity.name()} buttons={['list'/* @TODO add other links when entry is present*/]} />
+            <div className="view edit-view">
+                <ViewActions entityName={view.entity.name()} entry={entry} buttons={actions} />
 
                 <div className="page-header">
-                    <h1>{view.title() || "Edit one " +entityName}</h1>
+                    <h1>{view.title() || Inflector.singularize(entityName) + " edition"}</h1>
                     <p className="description">{view.description()}</p>
                 </div>
+
+                <div className="row form-horizontal" id="edit-view">
+                    <form className="col-lg-12 form-horizontal" onSubmit={this.save}>
+                        <FormFields
```

Comment:
in ng-admin, we have a component for a undetermined `Field`, but not for a list of `Fields`. Why did you choose another design here?